docs/section5/Rule5-26.md

```diff
@@ -18,79 +18,77 @@
 
-            **Rule Assertion - Component:**
+          - Case 1: For each subsurface that is vertical fenestration, if wall is exterior mixed type and the baseline requirements for residential and non-residential type SHGC for different window-wall-ratio are different: `if manual_review_flag AND ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ): UNDETERMINED`
 
-            - Case 1: For each subsurface that is vertical fenestration, if wall is exterior mixed type and the baseline requirements for residential and non-residential type SHGC for different window-wall-ratio are different: `if manual_review_flag AND ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ): UNDETERMINED`
+          - Case 2: Else if wall is exterior mixed type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_mixed ): PASS`
 
-            - Case 2: Else if wall is exterior mixed type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_mixed ): PASS`
+          - Case 3: Else if wall is exterior residential type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR RESIDENTIAL" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_res ): PASS`
 
-            - Case 3: Else if wall is exterior residential type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR RESIDENTIAL" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_res ): PASS`
+          - Case 4: Else if wall is exterior non-residential type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR NON-RESIDENTIAL" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_nonres ): PASS`
 
-            - Case 4: Else if wall is exterior non-residential type and vertical fenestration SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR NON-RESIDENTIAL" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_nonres ): PASS`
+          - Case 5: Else if wall is semi-exterior type and skylight SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "SEMI-EXTERIOR" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_semiheated ): PASS`
 
-            - Case 5: Else if wall is semi-exterior type and skylight SHGC matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "SEMI-EXTERIOR" ) AND ( subsurface_b.solar_heat_gain_coefficient == target_shgc_semiheated ): PASS`
-
-            - Case 6: Else: `else: FAIL and NUMBER_OF_FAIL_COMPONENTS++`
+          - Case 6: Else: `else: FAIL and NUMBER_OF_FAIL_COMPONENTS++`
 
 **Rule Assertion - RMR:**
```

Comment:
I don't think the entire rule should be tagged as "Undetermined" if at least one surface triggers such flag.

Reply:
Discussed at the RDS call, if one surface triggers a flag as mixed (res/non res) type, we will not be able to determine the actual res or non-res window-wall-ratio percentage, so the entire rule will be tagged as "undetermined".